Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM java:8-jre
+FROM openjdk:8-jre
 MAINTAINER Gabe Conradi <gabe@tumblr.com>
 
 # Solr cores should be stored in a volume, so we arent writing stuff to our rootfs
 VOLUME /opt/collins/conf/solr/cores/collins/data
 
 COPY . /build/collins
 RUN apt-get update && \
-    apt-get install --no-install-recommends -y openjdk-8-jdk zip unzip ipmitool && \
+    apt-get install --no-install-recommends -y openjdk-8-jdk openjdk-8-jdk-headless zip unzip ipmitool && \
```

Comment:
no, this is the JRE image. We install only the JDK for the duration of the build, then remove it